connectivity/check/deployment.go

```diff
@@ -875,18 +875,19 @@ func (ct *ConnectivityTest) deploy(ctx context.Context) error {
 			_, err = ct.clients.src.GetDeployment(ctx, ct.params.TestNamespace, echoExternalNodeDeploymentName, metav1.GetOptions{})
 			if err != nil {
 				ct.Logf("✨ [%s] Deploying echo-external-node deployment...", ct.clients.src.ClusterName())
-				containerPort := 8080
+				// in case if test concurrency is > 1 port must be unique for each test namespace
+				port := 8080 + ct.id
 				echoExternalDeployment := newDeployment(deploymentParameters{
 					Name:           echoExternalNodeDeploymentName,
 					Kind:           kindEchoExternalNodeName,
-					Port:           containerPort,
-					NamedPort:      "http-8080",
-					HostPort:       8080,
+					Port:           port,
```

Comment:
i probably missed it, but where is the corresponding change to adjust cilium network policies?

Reply:
Good question!
I've re-worker it a bit and replaced ConnectivityTest.id with [Parameters.ExternalDeploymentPort](https://github.com/cilium/cilium-cli/pull/2496/files#diff-6a57efc490ffd3a1c4a9274abc080ec97e305f89af38214d38f09736b445a2e7R68).
But I don't see any network policy that needs to be adjusted...